packages/traceloop-sdk/traceloop/sdk/utils/json_encoder.py

```
@@ -9,21 +9,31 @@ def default(self, o):
+                    return o.__class__.__name__
+
+                return result
+
+            except Exception:
+                pass
 
         if hasattr(o, "__class__"):
             return o.__class__.__name__
 
-        return super().default(o)
+        return super().default(o)
```

Ruff (0.15.12)

[warning] No newline at end of file

Add trailing newline

(W292)